pages/index.js

```diff
@@ -11,99 +14,125 @@ import IndexIsssueListItem from "../components/shared/IndexIssueListItem";
 function ServiceItem(props) {
   const { icon, content } = props;
   return (
-    <Col md={3} className="pb-4">
+    <Col className="px-3">
       <Row>
-        <Col xs="auto" className="pe-0">
-          <span style={{ color: "#00bfa5" }}>
-            <h1>{icon}</h1>
-          </span>
-        </Col>
-        <Col>
-          <h5>{content}</h5>
-        </Col>
+        <h1>{icon}</h1>
+      </Row>
+      <Row>
+        <p style={{ fontSize: "1.1rem" }}>{content}</p>
       </Row>
     </Col>
   );
 }
 
 export default function HomeView() {
-  const { projectIssues: projects } = useContext(IssuesContext);
+  const { issues } = useContext(IssuesContext);
+
+  const featuredIssues = [];
+
+  issues.map((issue) => {
+    if (issue.isFeatured) {
+      featuredIssues.push(issue);
+    }
+  });
 
   return (
     <>
       <Page fluid nav hideSeparator>
         <Row>
-          <Col xs={1} md={2}></Col>
-          <Col sm={12} md={9} lg={4} className="py-5 px-4 text-primary">
-            <h1 className="homepage-header">
-              <strong>Tech support for ATX mobility</strong>
-            </h1>
-            <p className="text-muted">
-              We build and buy technology to help city staff tackle
-              Austin&apos;s mobility challenges. We&apos;re obsessed with
-              efficiencies, open data, and delivering human-centered tech to
-              government services.
-            </p>
-          </Col>
-          <Col lg={4} className="my-auto d-none d-lg-block">
-            {/* hidden on xs and sm */}
-            <Image
-              fluid
-              src="/assets/sports.jpg"
-              alt="Illustration of a green bicycle"
-            />
-          </Col>
+          <Card className="hero-section p-0 h-100">
+            <Card.Body className="pt-5">
+              <Card.Title className="pt-4 text-center text-white fw-bold">
+                <h1 className="fw-bold" style={{ fontSize: "3.5rem" }}>
+                  Tech Support for ATX Mobility
+                </h1>
+              </Card.Title>
+              <Card.Text className="pt-3 text-center text-white">
+                <p style={{ fontSize: "1.75rem", fontWeight: 500 }}>
+                  We build and buy technology to help city staff <br></br>
+                  tackle Austin&apos;s mobility challenges.
+                </p>
+              </Card.Text>
+            </Card.Body>
+          </Card>
```

Comment:
Taking a quick pass through this - I think that these card components could be replaced with elements similar to the first example in [this Bootstrap 5 hero section](https://getbootstrap.com/docs/5.0/examples/heroes/) that also use [display headings](https://getbootstrap.com/docs/5.0/content/typography/#display-headings) for font size and uses elements/classes that don't have a border-radius OOTB.  

That way you don't have to work around all the built-in opinions of the Card components.

Reply:
TIL about display headings! Will definitely look into these. Thanks!